src/climate_learn/loaders.py

```diff
@@ -0,0 +1,350 @@
+
+
+def load_model_module(
+    task: str,
+    data_module: Union[DataModule, IterDataModule],
+    preset: Optional[str] = None,
+    model: Optional[Union[str, nn.Module]] = None,
+    model_kwargs: Optional[Dict[str, Any]] = None,
+    optim: Optional[Union[str, torch.optim.Optimizer]] = None,
+    optim_kwargs: Optional[Dict[str, Any]] = None,
+    sched: Optional[Union[str, LRScheduler]] = None,
+    sched_kwargs: Optional[Dict[str, Any]] = None,
+    train_loss: Optional[Union[str, Callable]] = None,
+    val_loss: Optional[Iterable[Union[str, Callable]]] = None,
+    test_loss: Optional[Iterable[Union[str, Callable]]] = None,
+    train_target_transform: Optional[Union[str, Callable]] = None,
+    val_target_transform: Optional[Iterable[Union[str, Callable]]] = None,
+    test_target_transform: Optional[Iterable[Union[str, Callable]]] = None,
+):
+    # Temporary fix, per this discussion:
+    # https://github.com/aditya-grover/climate-learn/pull/100#discussion_r1192812343
+    if data_module.get_lat_lon() is None:
+        raise RuntimeError("Data module has not been set up yet.")
+    # Load the model
+    if preset is None and model is None:
+        raise RuntimeError("Please specify 'preset' or 'model'")
+    elif preset:
+        print(f"Loading preset: {preset}")
+        model, optimizer, lr_scheduler = load_preset(task, data_module, preset)
+    elif isinstance(model, str):
+        print(f"Loading model: {model}")
+        model_cls = MODEL_REGISTRY.get(model, None)
+        if model_cls is None:
+            raise NotImplementedError(
+                f"{model} is not an implemented model. If you think it should be,"
+                " please raise an issue at"
+                " https://github.com/aditya-grover/climate-learn/issues."
+            )
+        model = model_cls(**model_kwargs)
+    elif isinstance(model, nn.Module):
+        print("Using custom network")
+    else:
+        raise TypeError("'model' must be str or nn.Module")
+    # Load the optimizer
+    if preset is None and optim is None:
+        raise RuntimeError("Please specify 'preset' or 'optim'")
+    elif preset:
+        print("Using preset optimizer")
+    elif isinstance(optim, str):
+        print(f"Loading optimizer {optim}")
+        optimizer = load_optimizer(model, optim, optim_kwargs)
+    elif isinstance(optim, torch.optim.Optimizer):
+        print("Using custom optimizer")
+    else:
+        raise TypeError("'optim' must be str or torch.optim.Optimizer")
+    # Load the LR scheduler
+    if preset is None and sched is None:
+        raise RuntimeError("Please specify 'preset' or 'sched'")
+    elif preset:
+        print("Using preset learning rate scheduler")
+    elif isinstance(sched, str):
+        print(f"Loading learning rate scheduler: {sched}")
+        lr_scheduler = load_lr_scheduler(sched, optimizer, sched_kwargs)
+    elif isinstance(sched, LRScheduler._LRScheduler):
```

Comment:
I think based on the import this can be changed from `LRScheduler._LRScheduler` to `LRScheduler`.